app/views/assignments/_list_manage.html.erb

```diff
@@ -7,77 +7,105 @@
-                                { controller: controller,
-                                  action: action,
-                                  assignment_id: assignment },
-                                class: 'assignment_list' %>
-                  <% else %>
-                    <%= link_to assignment.short_identifier + ': ' + assignment.description,
-                                { controller: controller,
-                                  action: action,
-                                  assignment_id: assignment },
-                                class: 'assignment_list'%>
-                  <% end %>
+                  <%= link_to assignment.short_identifier + ': ' + assignment.description,
+                              { controller: controller,
+                                action: action,
+                                assignment_id: assignment },
+                              class: 'assignment_list' %>
                 <% end %>
-              </td>
+              <% end %>
+            </td>
 
-              <td>
-                <%= l(assignment.due_date, format: :long_date ) %>
-              </td>
+            <td>
+              <%= l(assignment.due_date, format: :long_date) %>
+            </td>
+
+            <td>
+              <ul>
+                <% if assignment.submission_rule.can_collect_now? %>
+                  <li>                <span id='collected_submissions_<%= assignment.short_identifier %>'>
 
-              <td>
-                <span id='collected_submissions_<%= assignment.short_identifier %>'>
-                  <% if assignment.submission_rule.can_collect_now? %>
                     <%= link_to t('submissions_collected',
                                   count: assignment.groupings
-                                                   .where(is_collected: true)
-                                                   .size,
+                                      .where(is_collected: true)
+                                      .size,
                                   size: assignment.groupings.size),
-                                  controller: :submissions,
-                                  action: :browse,
-                                  assignment_id: assignment.id %>
-                  <% end %>
-                </span>
-                <span id='class_average__<%= assignment.short_identifier %>'>
-                  <% if !assignment.results_average.nil? %>
+                                controller: :submissions,
+                                action: :browse,
+                                assignment_id: assignment.id %>
+
+                </span></li>
+                <% end %>
+                <% if !assignment.results_average.nil? %>
+                  <li><span id='class_average__<%= assignment.short_identifier %>'>
+
                     <%= t(:class_average,
                           results_average: assignment.results_average) %>
```

Comment:
I think it'd be nice to have a % sign afterwards (assuming it's a percentage average). @david-yz-liu : thoughts?